src/utils/slug.js

```diff
@@ -1,8 +1,12 @@
 // Convert a string to a URL-friendly slug
 module.exports.makeSlug = string => {
-  return string
-    .toLowerCase()
-    .replace("&", "and")
-    .replace(/[^\w\- ]/g, "")
-    .replace(/ +/g, "-")
+  return (
+    string
+      .toLowerCase()
+      .trim()
+      .replace("&", "and")
+      // remove all non-alphanumeric chars and not-defined chars (+, -, etc)
+      .replace(/[^\w\-+ ]/g, "")
```

Comment:
I think a word replace might work better here, like we did with "&". Here are all the examples from our DB today:

* "100+ Accelerator" -> `100plus-acelerator` vs `100+-accelerator`
* "Terra Food + AG Tech Accelerator" -> `terra-food-plus-ag-tech-accelerator` vs `terra-food-+-ag-tech-accelerator`
* "Planet+" -> `planetplus` vs `planet+`